shared/constants/fs.js

```diff
@@ -1151,8 +1151,14 @@ export const makeActionsForShowSendAttachmentToChat = (
   ),
 ]
 
-export const splitFileNameAndExtension = (fileName: string) =>
-  ((str, idx) => [str.slice(0, idx), str.slice(idx)])(fileName, fileName.lastIndexOf('.'))
+export const splitFileNameAndExtension = (fileName: string) => {
+  const idx = fileName.lastIndexOf('.')
+  if (idx === -1 || fileName.length - idx > 10) {
+    return [fileName, '']
+  } else {
+    return [fileName.slice(0, idx), fileName.slice(idx)]
+  }
+}
```

Comment:
It feels a bit weird to me to returns an array with second string being empty. Probably better to have an object or I.Record type for this. That being said, this is only used inside the component, so I think it should stay in that file and the function interface would not matter as much.